### DISCUSSION TITLE
Integration with Monk's Token Bar and/or Epic Rolls

### DISCUSSION THREAD

**Discussion by spitfire1081** [Saibot393/perceptive#77] [Ideas]
I use to ask player checks using Monks Token Bar or Epic Rolls. It would be great if Perceptive could detect the Monk's checks, even to make better automation for secret doors (with blind rolls from trigger tiles, for example).

**Comment by Saibot393** [comment]
I will look into it and see what can be done

**Reply by spitfire1081** [reply]
Thank you for your reply.

**Reply by Saibot393** [reply]
I looked into it, the way monks rolls the checks is pretty much completely incompatible with the way perceptive currently detects skill checks. I will probably be able to do it when i have a lot of time to implement some custom code for it, i can't give an ETA for it. While it can in theory be done relatively easily for epic rolls there is another problem that would require a lot of additional code to prevent registering double rolls. I will take a look at this at a lter time.
